src/browser/of_events.ts

```diff
@@ -42,6 +42,15 @@ class OFEvents extends EventEmitter {
 
         return super.emit(route, ...data);
     }
+
+    public SUBSCRIBER: StringMap = {
```

Comment:
Regarding static, note the last line:
```js
export default new OFEvents();
```
So the 14 files that import this only have the instantiated object handy and not the constructor. Instead of `ofEvents.SUBSCRIBER.ADDED` we would have to have `ofEvents.constructor.SUBSCRIBER.ADDED`. This does not IMHO to enhance readability.